features/admin.applications.v1/components/forms/general-details-form/general-details-form.scss

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+.application-general-discoverable-groups {
```

Comment:
Let's add a parent class to application general sections and wrap discoverable-groups inside it, so that any other application general tab related styles can go under the parent class

eg:

.application-general-settings {
   .discoverable-groups {
   }

   /* any other styles */
}

Reply:
addressed by https://github.com/wso2/identity-apps/pull/7399/commits/dc902a134667e113b931b40d444624c884e6ddbb